src/protoprimer/main/protoprimer/primer_kernel.py

```diff
@@ -2541,6 +2639,56 @@ def _eval_state_once(
         return state_env_local_venv_dir_abs_path_eval_finalized
 
 
+# noinspection PyPep8Naming
+class Bootstrapper_state_env_local_log_dir_abs_path_eval_finalized(
```

Comment:
TODO: Consider gconf/lconf overrides. There are many cases when this might be a generic pattern.

Reply:
Not this time...